README.jp.md

```diff
@@ -0,0 +1,805 @@
+_Only the original [README](README.md) is guaranteed to be up-to-date._
+
+# scrcpy (v1.17)
+
+[他の言語で読む](#translations)
+
+このアプリケーションはUSB(もしくは[TCP/IP経由][article-tcpip])で接続された
```

Comment:
Please remove all line breaks.

It's OK to have them in English, however in other languages these will leave extra spaces.

Reply:
I think there is no problem in Japanese.

Reply:
Look at the preview, there are spaces all over the places.

![image](https://user-images.githubusercontent.com/1330321/111867710-c5294e00-89b0-11eb-9f1e-a7a6f0634091.png)

Reply:
Thank you, now I understand.